assets/src/scripts/_files-list.js

```diff
@@ -5,43 +5,37 @@ import { approvedFiles } from "./_signals";
 
 const fileList = () => {
   const container = document.getElementById("filesList");
-  const children = [...container.children];
+  const fileListItems = [...container.querySelectorAll("li")];
 
   // add click handler to each list item
-  children.forEach((el) => {
-    const fileName = el.id;
+  fileListItems.forEach((el) => {
+    const fileName = el.getAttribute("data-file-name");
     const metadata = outputs.get(fileName);
 
     // get the URL and strip off the leading #
-    const url = el.firstElementChild.href.replace("#", "");
+    const url = el.querySelector("a").href.replace("#/", "");
 
     // toggle selected state for the file list
     el.addEventListener("click", () => {
       handleFileClick({ fileName, metadata, url });
 
       // clear selected class from all items in the list
-      children.forEach((e) => e.classList.remove("selected"));
+      fileListItems.forEach((e) => e.classList.remove("bg-blue-50"));
```

Comment:
I would prefer a semantic class name here rather than concrete style?

Reply:
Yes, but if we can ignore this for now, I'm going to add better styles for the various states in the future.